packages/netdog/netdog.spec

```
@@ -0,0 +1,133 @@
+%global _cross_first_party 1
+%undefine _debugsource_packages
+
+Name: %{_cross_os}netdog
+Version: 0.1.0
+Release: 0%{?dist}
+Summary: Bottlerocket network configuration helper
+License: Apache-2.0 OR MIT
+URL: https://github.com/bottlerocket-os/bottlerocket
+
+Source0: netdog-tmpfiles.conf
+
+Source10: run-netdog.mount
+Source11: write-network-status.service
+Source12: generate-network-config.service
+Source13: disable-udp-offload.service
+
+Source20: 00-resolved.conf
+
+BuildRequires: %{_cross_os}glibc-devel
+Requires: %{_cross_os}netdog
+Requires: (%{_cross_os}netdog-systemd-networkd if %{_cross_os}image-feature(systemd-networkd))
+Requires: (%{_cross_os}netdog-wicked if %{_cross_os}image-feature(no-systemd-networkd))
+
+%description
+%{summary}.
+
+%package -n %{_cross_os}netdog-common
```

Prefer the shorthand syntax for subpackages that suffix the main package name:
```suggestion
%package common
```